dashboards-observability/public/components/explorer/hooks/use_fetch_events.ts

```diff
@@ -28,6 +28,7 @@ import {
 } from '../slices/field_slice';
 import PPLService from '../../../services/requests/ppl';
 import { IField } from 'common/types/explorer';
+import { any, array } from 'joi';
```

Comment:
I think this import shouldn't be here. "any" datatype doesn't need an explicit import.

Reply:
Yeah okay, i'll remove it